src/ray/core_worker/fiber.h

```diff
@@ -18,6 +18,8 @@
 #include <chrono>
 
 #include "ray/util/logging.h"
+#include "ray/util/macros.h"
```

Comment:
Is this needed

Reply:
needed for RAY_UNUSED. Not sure why it did not complain before.